code/_macros.dm

```diff
@@ -1,6 +1,8 @@
 #define Clamp(x, y, z) 	(x <= y ? y : (x >= z ? z : x))
 #define CLAMP01(x) 		(Clamp(x, 0, 1))
 
+#define span(class, text) ("<span class='[class]'>[text]</span>")

```

Comment:
until now ive used span like span("class", "text")

Won't this macro result in "<span class='"class"'>[text]</span>"
those extra quotes could be a problem

Since its a define the quotes around the input may be no longer needed. would that work?